config/default-settings.json

```diff
@@ -1,12 +1,12 @@
 {
-  "motd": "A Minecraft Server \nRunning flying-squid",
+  "motd": "A Minecraft Server \nRunning Flying Squid",
   "port": 25565,
-  "max-players": 10,
+  "max-players": 25,
   "online-mode": true,
   "logging": true,
-  "gameMode": 1,
-  "difficulty": 1,
-  "worldFolder":"world",
+  "gameMode": 0,
```

Comment:
Survival mode is practically unplayable with flying-squid currently.

Reply:
Fixed